static/opensearch.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
```

Comment:
is HTTP -> HTTPS correct change for namespace?

Reply:
Well, I forgot that XML namespaces are not URLs. It is a mistake of course. Why they put `http://` there is the first place? It is plain confusing.